src/main/java/com/lowdragmc/mbd2/api/pattern/predicates/SimplePredicate.java

```diff
@@ -66,8 +74,8 @@ public class SimplePredicate implements IAutoPersistedSerializable, IConfigurabl
     @Configurable(name = "config.block_pattern.predicate.previewCount", tips = { "config.block_pattern.predicate.previewCount.tooltip.0", "config.block_pattern.predicate.previewCount.tooltip.1" })
     @NumberRange(range = {-1, Integer.MAX_VALUE})
     public int previewCount = -1;
-    @Configurable(name = "config.block_pattern.predicate.disableRenderFormed", tips = "config.block_pattern.predicate.disableRenderFormed.tooltip")
-    public boolean disableRenderFormed = false;
+    @Configurable(name = "config.block_pattern.predicate.disableRenderFormed", tips = "config.block_pattern.predicate.disableRenderFormed.tooltip", subConfigurable = true)
+    public ToggleRenderedForm disableRenderFormed = new ToggleRenderedForm(false);
```

Comment:
could you make it a new feautre instead of enhance the existing feautre?
```java
@Configurable(name = "config.block_pattern.predicate.disableRenderFormed", tips = "config.block_pattern.predicate.disableRenderFormed.tooltip")
public boolean disableRenderFormed = false;
@Configurable(name = "config.block_pattern.predicate.disableRenderFormed", tips = "config.block_pattern.predicate.disableRenderFormed.tooltip", subConfigurable = true)
public ToggleRenderedForm replaceBlockWhenFormed = new ToggleRenderedForm(false);
```